packages/perseus/src/types.ts

```diff
@@ -92,7 +92,6 @@ export type ChangeHandler = (
         currentValue?: string;
         // perseus-all-package/widgets/dropdown.jsx
         selected?: number;
-        // perseus-all-package/widgets/transformer.jsx
         tools?: ReadonlyArray<any>;
         transformations?: ReadonlyArray<any>;
         highlights?: SerializedHighlightSet;
```

Comment:
**question**: The way the deleted comment worked is that it signified the following change handler props were from the `transformer` widget.  You might be able to delete these three args now that `transformer` is gone!

Reply:
It's no harm leaving it, but _if_ we did still use it, I suspect something somewhere would crash, yes.